detection_rules/packaging.py

```diff
@@ -135,6 +136,22 @@ def _package_notice_file(save_dir):
             lines = lines + commented_notice + [' */', '']
             f.write('\n'.join(lines))
 
+    def _package_index_file(self, save_dir):
+        """Convert and save index file with package."""
+        e_license = ['/*', '\n'.join(' * ' + line.lstrip('# ') for line in LICENSE_HEADER.splitlines()), ' */', '']
+        comments = [
+            '// Auto generated file from scripts/regen_prepackage_rules_index.sh',
+            '// Do not hand edit. Run that script to regenerate package information instead',
+            ''
+        ]
+        rule_imports = [f"import rule{i} from './{os.path.splitext(os.path.basename(r.path))[0] + '.json'}';"
+                        for i, r in enumerate(self.rules, 1)]
+        const_exports = ['export const rawRules = ['] + [f"  rule{i}," for i in range(1, len(self.rules))] + ['];', '']
```

Comment:
```suggestion
        const_exports = ['export const rawRules = ['] 
        const_exports.extend(f"  rule{i}," for i in range(1, len(self.rules)))
        const_exports.append("];")
        const_exports.append(" ")
```

Reply:
This is dropping one rule:
```python
>>> len(range(1, 10))
9
```

Reply:
Can you sort  the rules by some predicatable way? At first I thought sorting by file name would be best, but now I'm thinking that created_date would produce a better diff. Either is fine, as long as we pick something more deterministic

Reply:
good call